### DISCUSSION TITLE
Why do React Native packages compile with SyntaxError: "Support for the experimental syntax 'jsx' isn't currently enabled"?

### DISCUSSION THREAD

**Discussion by restarajat** [necolas/react-native-web#2470] [Help] [answered]
In my project, I'm using the reanimate library to add animations, but I'm receiving an error

![screenshot-localhost_3001-2023 01 23-10_12_31](https://user-images.githubusercontent.com/118878172/213967505-c99a5d3c-9e98-45d0-861d-1698f0dd664e.png)

package.json 

![Screenshot 2023-01-23 at 10 37 49 AM](https://user-images.githubusercontent.com/118878172/213969610-073e562c-0826-4c17-8e4b-4c83367f986a.png)

webpack.config.js

![Screenshot 2023-01-23 at 10 42 23 AM](https://user-images.githubusercontent.com/118878172/213969947-a749310a-c9e2-4a6e-ab61-4f74284ced63.png)

**Comment by lzach83** [comment]
I'm having the same error

**Comment by necolas** [comment, marked as answer]
There's a comment on L15 of your webpack config that says "Add every directory that needs to be compiled by Babel..." And before that is a comment block with more explanations. And the error messages you're getting are telling you the same thing. But you haven't added any RN packages to your babel loader config which is why the modules still have JSX syntax in them.

If after reading documentation you still have problems configuring your bundler, I suggest using Expo which will take care of this work out-of-the-box.
